src/components/molecules/PayQR/PayQR.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import QRCode from '../../atoms/QRCode'
+
+import coffeeCup from '../../../assets/images/favicon.svg'
+import { toast } from 'react-toastify'
+import { useFormats } from '../../../utils/useFormats'
+import Typography from '../../atoms/Typography'
+import { ButtonWrapper, PayQR, QRWrapper } from './PayQR.styles'
+import { useUser } from '../../../context/User'
+import Button from '../../atoms/Button'
+import { useRemove } from '../../../utils/useRemove'
+import { ButtonEnum } from '../../../models/Enums/Button'
+
+
+const PayQRComponent = () => {
+  const { formatCurrency } = useFormats()
+  const [viewPIX, setViewPIX] = useState<string>('')
+  const { state } = useUser()
+  const { removePaymentValue } = useRemove()
+
+  const clearPaymentValue = () => {
+    removePaymentValue()
+    toast.success('Suas compras foram pagas! Você não está com um peso na consciência, certo?')
+  }
+
+  return (
+    <PayQR>
+      <Typography as="h2" centralized>QR com {formatCurrency(state.paymentValue)}</Typography>
+      <QRWrapper
+        onClick={() => { navigator.clipboard.writeText(viewPIX), toast.success('PIX copiado com sucesso!') }}
+      >
+        <QRCode
+          pixkey="26442024000194"
```

Comment:
Tornar valores dinâmicos recebidos por parâmetro ou utils. Componente com propósito único. Se for específico, renomear para especificar melhor o seu uso.

Reply:
Solução tomada: armazenamento das informações atraveś do .env